spec/mailers/claim_mailer_spec.rb

```diff
@@ -74,6 +74,18 @@
           expect(mail.body.encoded).to include("not been able to approve")
         end
       end
+
+      describe "#update_after_three_weeks" do
```

Comment:
Can we get a test to check that an approved claim doesn't do anything?

Reply:
Just looking into how to implement this test makes me wonder if the claim mailer is the right place for a guard to live? It feels like it's not the concern of the mailer to check the current state of the claim - it's been told to send an email, so it should.

Thoughts?

Reply:
Yeah, you're probably right. Actually, thinking about it, the guard probably doesn't make too much sense. We're queuing the emails at 8am, and they'll get rattled through pretty quickly, so the odds on someone checking and approving claims at this time are pretty slim